cmd/bb_replicator/main.go

```diff
@@ -53,7 +53,9 @@ func main() {
 			configuration.Replicator,
 			source.BlobAccess,
 			sink,
-			blobstore_configuration.NewCASBlobReplicatorCreator(grpcClientFactory))
+			blobstore_configuration.NewCASBlobReplicatorCreator(grpcClientFactory),
+			"cas",
```

Comment:
Probably better to call `blobAccessCreator.GetStorageTypeName()` here, to make sure it doesn't go out of sync?